hawkauthlib/__init__.py

```diff
@@ -52,13 +52,23 @@
 
 
 @utils.normalize_request_object
-def sign_request(request, id_, key, algorithm=None, params=None):
+def sign_request(request, id_, key, include_payload_hash=True, algorithm=None, params=None):
```

Comment:
Since this is intended to be called by clients, it seems useful to expose the ability to calculate a payload has on the client while we're here.